src/collections/_documentation/development/sdk-dev/event-payloads/exception.md

```diff
@@ -258,7 +285,13 @@ Generic unhandled crash:
 
 ```json
 {
-    "type": "generic",
-    "handled": false
+  "exception": {
```

Comment:
This example was (confusingly) just a mechanism. The full example would be:

```json
{
  "exception": {
    "values": [
      {
        "type": "Error",
        "value": "An error occurred",
        "mechanism": {
          "type": "generic",
          "handled": false
        }
      }
    ]
  }
}
```

Reply:
Thanks, updated.